elements/pfe-navigation/src/pfe-navigation.js

```diff
@@ -1039,6 +1052,12 @@ class PfeNavigation extends PFElement {
     }
   }
 
+  // close menu when overlay is clicked
+  _overlayClickHandler() {
+    const openToggleId = this.getAttribute(`${this.tag}-open-toggle`);
+    this._changeNavigationState(openToggleId, "close");
```

Comment:
I think this click handler needs to be a little more robust like we talked about
```js
  // close menu when overlay is clicked
  _overlayClickHandler() {
    const openToggleId = this.getAttribute(`${this.tag}-open-toggle`);
    if (this.isSecondaryLinksSectionCollapsed()) {
      // Mobile
      this._changeNavigationState('mobile__button', "close");
    }
    else if (this.isMobileMenuButtonVisible()) {
      // Tablet-ish
      // @todo if it's a child of main menu (e.g. openToggleId.startsWith("main-menu")) close mobile__button
      // Else close openToggleId
    } else {
      // Desktop
      this._changeNavigationState(openToggleId, "close");
    }
  }
```